index.html

```diff
@@ -326,29 +326,29 @@ <h3 class="skills-heading">
         </section>
 
         <!-- ======================== Contact Page ======================== -->
-        <a name="contact-page"></a>
         <section class="contact-form-page">
+            <a name="contact-page"></a>
```

Comment:
- [ ] I also noticed that you have this empty `<a>` tag inside your Works, About, and Contact sections. I don't know the purpose of it, but assistive technologies such as screen readers can get misleading information from this tag, which is suggesting that there is a link without a reference. Please avoid using empty elements! 🤓 🔧

Reply:
I have done this to access the Works, About, and Contact sections from the nav bar. If there is a better approach for doing so, kindly enlighten me with it.

https://user-images.githubusercontent.com/122236545/215921508-5664e43d-b10b-4bfa-a1d3-198a1d4e83d8.mp4

Reply:
Hi @sameedmohsin28, sorry for the late response. When you use a hash value like `#random_text` inside the `href` attribute of an anchor `<a>` tag, you're pointing that link to the element with this id, in this case: `random_text` so you don't need empty `<a>` tags, instead, just define the `id` of the container element as the one you have in the `href` and the link will work fine! 🤓 🔧